Library/Formula/hashpump.rb

```diff
@@ -10,13 +10,23 @@ class Hashpump < Formula
     sha256 "cf4a9cee9d04bef03016073dc0e516e5349861691ad80bae1706cbd846b03920" => :mountain_lion
   end
 
+  option "with-python", "Build with Python bindings"
+
   depends_on "openssl"
+  depends_on :python => :optional
 
   def install
     bin.mkpath
     system "make", "INSTALLLOCATION=#{bin}",
                    "CXX=#{ENV.cxx}",
                    "install"
+
+    if build.with? "python"
+      # The Python bindings require the headers, which are named generically and
+      # can conflict with other package's headers.
```

Comment:
So it is. That's a really weird way to install a Python extension! Installing to libexec seems correct.

Since this supports Python 3, you can even add `option "without-python"` and `depends_on :python3 => :optional` and wrap this in Language::Python.each_python if you want.